problems/data_science/chipmunks/data_science_project_night_4_18_19.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
I think the whole idea is very cute :D ... the story line guides you through a natural progression.
However from my experience this would be a lot of new concepts to grasp for people who would be seeing a dataframe for the first time. As you know there is an hour time limit for the whole event, there would be a lot to unpack.
I would like to see this being done over multiple project nights - where instead of rushing to the end, we give ample time to the returning groups to work through it to the end.